### PR TITLE
Minor refinements

### PR DESCRIPTION
**Pull request by MahdiBM**

### CODE REVIEW

**Review by 0xTim — APPROVED** [review]

File: Dockerfile [hunk 1]
```diff
@@ -17,37 +17,48 @@ WORKDIR /build
...
 RUN find -L "$(swift build --package-path /build -c release --show-bin-path)/" -regex '.*\.resources$' -exec cp -Ra {} ./ \;
 
-# Move /Templates/Penny directory to staging area
-RUN mkdir -p ./Templates && mv /build/Templates/Penny ./Templates/Penny && chmod -R a-w ./Templates/Penny;
-
 # ================================
 # Run image
 # ================================
-FROM ubuntu:jammy
+FROM ubuntu:latest
```
Comment (gwynne) [inline Dockerfile:48]: This is not safe. Swift does not officially support post-Jammy Ubuntu yet.

File: Dockerfile [hunk 2]
```diff
@@ -17,37 +17,48 @@ WORKDIR /build
...
 COPY . .
 
-# Build everything, with optimizations
-RUN swift build -c release --static-swift-stdlib --product Penny
+# Build everything, with optimizations, with static linking, and using jemalloc
+# N.B.: The static version of jemalloc is incompatible with the static Swift runtime.
+RUN swift build -c release \
+      --target Penny \
+      --static-swift-stdlib \
+      -Xlinker -ljemalloc
```
Comment (MahdiBM) [inline Dockerfile:31]: `--target Penny` instead of `--product` because i removed the product as it seemed unnecessary.

File: Dockerfile [hunk 3]
```diff
@@ -59,12 +70,14 @@ WORKDIR /app
...
+
 # Ensure all further commands run as the vapor user
 USER vapor:vapor
 
 # Let Docker bind to port 8080
 EXPOSE 8080
 
-# Start the Vapor service when the image is run, default to listening on 8080 in production environment
-ENTRYPOINT [ "./Penny" ]
-CMD [ "serve", "--env", "production", "--hostname", "0.0.0.0", "--port", "8080" ]
```
Comment (MahdiBM) [inline Dockerfile:70]: `CMD` is useful for Vapor apps but doesn't do anything for Penny as it's not a Vapor app.

File: Lambdas/GHHooks/EventHandler/Handlers/IssueHandler.swift [hunk 4]
```diff
@@ -228,12 +228,10 @@ private enum Status: String {
...
-        if issue.state_reason == .not_planned {
-            if issue.knownLabels.contains(.duplicate) {
-                self = .duplicate
-            } else {
-                self = .notPlanned
-            }
+        if issue.knownLabels.contains(.duplicate) {
+            self = .duplicate
+        } else if issue.state_reason == .not_planned {
+            self = .notPlanned
```
Comment (MahdiBM) [inline Lambdas/GHHooks/EventHandler/Handlers/IssueHandler.swift:234]: if `duplicate` label exists then it's always a duplicate issue. Regardless if someone closed the issue as "Not Planned" or not.

File: Dockerfile (hunk 1 above)
Comment (MahdiBM) [reply Dockerfile:48]: Hmm i believe i only copied it from the new template Dockerfile... i was thinking the same too about Swift support 🤔

Comment (MahdiBM) [reply Dockerfile]: reverted

**Review by gwynne — APPROVED** [review]

File: Lambdas/GitHubAPI/openapi-generator-config.yml [hunk 5]
```diff
@@ -1,3 +1,4 @@
 generate:
   - types
   - client
+accessModifier: package
```
Comment (MahdiBM) [inline Lambdas/GitHubAPI/openapi-generator-config.yml:4]: They changed the default to `internal` before the 1.0.0 release.

**Comment by MahdiBM** [comment]
Nightly failures are due to the upcoming features i enabled (internal import by default).
I'll take care of it when it's in a tagged-nightly and is more stable.

**Pull request merged**